### DISCUSSION TITLE
Filament retracting too much when using Octolapse

### DISCUSSION THREAD

**Discussion by iotacb** [Jyers/Marlin#605] [Q&A] [answered]
When I try to print a timelapse using Octolapse and the latest firmware version (1.3.1) the filament gets retracted way too much when a snapshot is taken. Then the print fails because the is no filament to extrude. I have the same issue with my Sidewinder X1 and custom firmware. I don't know if it's my bad or something else is wrong.

**Comment by Jyers** [comment, marked as answer]
This is most likely an issue with your octolapse settings, as it adds the gcode for the retractions.
